client/hudwidget.cpp

```diff
@@ -1630,6 +1629,15 @@ QString popup_terrain_info(struct tile *ptile)
   return ret;
 }
 
+/**
+   Show music info
```

Comment:
New functions use this style:
```
/**
 * Comment
 */
```
The old style is there only because I didn't dare mass-replacing it with regex.